fieldservice_recurring/views/fsm_recurring.xml

```diff
@@ -40,6 +40,7 @@
                     <field name="state" widget="statusbar"/>
                 </header>
                 <sheet>
+                    <div class="oe_button_box" name="button_box"/>
```

Comment:
@kongrattapong, I think you will need to rebase your code and squash commits to prevent duplicating this in the history